Payment/Expense/index.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { issuefab } from "../../index"
+
+describe("Payment.Expense", () => {
+	const expense: issuefab.Payment.Expense = {
+		type: "expense",
+		limit: [10, "EUR"],
+	}
+	it("is", () => {
+		expect(issuefab.Payment.Expense.Creatable.is(expense)).toEqual(true)
```

Comment:
`issuefab.Payment.Expense.Creatable.is(expense)` is tested in the Creatable.spec.ts file. this can safely be removed